source/mainnet/smart-contracts/tutorials/sft-minting/build-smart-contract.rst

```diff
@@ -4,11 +4,11 @@
 Smart contract modifications
 ============================
 
-You are using the example contract from Concordium's examples and it’s ready to use. If you want to use it as is you can do it for your project. But in this tutorial you will add a couple of things and update some functions to give more flexibility.
+In this tutorial, you will use the example ``cis2-nft`` contract template from Concordium, which you can initialize using the cargo-concordium tool. In this tutorial you will add a couple of things and update some functions to give more flexibility to the template.
```

Comment:
Suggestion: would it make sense to provide the command how to get the template : `cargo concordium init` (if it is not mentioned somewhere else in this tutorial already).